packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -198,18 +223,31 @@ const getStyles = (theme: GrafanaTheme2) => {
         outline: `1px solid ${theme.colors.action.focus}`,
       },
     }),
+    loadingBarContainer: css({
+      position: 'absolute',
+      top: 0,
+      width: '100%',
+    }),
     content: css({
       label: 'panel-content',
       width: '100%',
-      contain: 'strict',
       flexGrow: 1,
+      overflow: 'auto',
```

Comment:
we cannot have overflow: auto, it will cause tooltips and dropdowns to have to use portalling (and has other issues).

can you describe the reasons for the change?

Reply:
We added it in favor of contain: strict we had to remove because it broke the layout of all panels when they are in a flex layout.

I'll look into different solutions.